tests/Agent/IntegrationTests/ApplicationHelperLibraries/NetCore3Collectible/CollectibleInstrumented.cs

```diff
@@ -11,9 +13,15 @@ public class CollectibleInstrumented
     {
         [Trace]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public IDistributedTracePayload GetDistributedTracePayload()
+        public Dictionary<string, string> InsertDistributedTraceHeaders()
         {
-            return NewRelic.Api.Agent.NewRelic.GetAgent().CurrentTransaction?.CreateDistributedTracePayload();
+            var headers = new Dictionary<string, string>();
+            IAgent agent = NewRelic.Api.Agent.NewRelic.GetAgent();
+            ITransaction currentTransaction = agent.CurrentTransaction;
+            var tooMuchWorkToGetAtDataCallback = new Action<Dictionary<string, string>, string, string>((carrier, key, value) => { carrier[key] = value; });
```

Comment:
interesting variable name

Reply:
I'm not a huge fan of the new DT api. I will modify this to have less snark :)

Reply:
Updated